api/db-cdr/vars.env

```diff
@@ -2,8 +2,8 @@ DB_CONNECTION_STRING=jdbc:mysql://db/cdr
 DB_DRIVER=com.mysql.jdbc.Driver
 DB_HOST=db
 DB_PORT=3306
-CDR_DB_NAME=cdr
-PUBLIC_DB_NAME=public
+CDR_DB_NAME=cdr20180702
```

Comment:
I don't think this change should be checked in; our scripts are still using "cdr" and "public" by default, right?

Reply:
Good catch. Thanks. I didn't see that one.

Reply:
Changed it. I generated a new data and was testing it.